llvm/test/Transforms/BDCE/dead-uses.ll

```diff
@@ -9,8 +9,7 @@ declare <2 x i32> @llvm.fshr.v2i32(<2 x i32>, <2 x i32>, <2 x i32>)
 ; First fshr operand is dead.
 define i32 @pr39771_fshr_multi_use_instr(i32 %a) {
 ; CHECK-LABEL: @pr39771_fshr_multi_use_instr(
-; CHECK-NEXT:    [[X:%.*]] = or i32 [[A:%.*]], 0
```

Comment:
Can you change the `or` operand to  `1` (or something else that doesn't optimize away) instead? Dropping it is not really the point of this tests.